packages/@react-spectrum/s2/stories/DateField.stories.tsx

```diff
@@ -17,8 +17,12 @@ import type {Meta} from '@storybook/react';
 const meta: Meta<typeof DateField> = {
   component: DateField,
   parameters: {
-    layout: 'centered'
-  }
+    layout: 'centered',
+    chromatic: {
+      disableSnapshot: true
```

Comment:
Should we remove these autogenerated stories for components we haven't worked on yet?

Reply:
good idea